src/REAXFF/fix_reaxff_species.cpp

```diff
@@ -45,7 +47,8 @@ FixReaxFFSpecies::FixReaxFFSpecies(LAMMPS *lmp, int narg, char **arg) : Fix(lmp,
 {
   if (narg < 7) error->all(FLERR, "Illegal fix reaxff/species command");
 
-  force_reneighbor = 0;
+  force_reneighbor = 1;
```

Comment:
Sure, it could be written that way. The reneighboring triggered by molecule deletion is technically redundant to the reneighboring settings already 'enforced' by fix reaxff/species. However, I like the idea of retaining it in the deletion routine for clarity, in case someone copies the code block elsewhere. I use quotations on 'enforced', because I think reneighboring settings can be easily modified by subsequent commands, perhaps rendering the reaxff/species calculation incorrect, but this is not checked for